Cargo.toml

```diff
@@ -14,12 +14,18 @@ edition = "2021"
 exclude = ["/.github/*", "/.travis.yml", "/appveyor.yml"]
 
 [dependencies]
+futures = { version = "0.3", optional = true }
```

Comment:
It's not worth introducing this crate only for `BoxFuture` (which is just a type alias).